tests/php/unit/Controller/BookingControllerTest.php

```diff
@@ -151,23 +167,108 @@ public function testGetBookableSlotsInvalidTimezone(): void {
 		$this->controller->getBookableSlots($apptConfg->getId(), $start, 'Hook/Neverland');
 	}
 
-	public function testGetBookableSlotsDatesInPast(): void {
-		$start = time();
-		$fakeFutureTimestamp = time() + (100 * 24 * 60 * 60);
+	public function testGetBookableSlotsTimezoneIdentical(): void {
+
+		$now = (new DateTime('2024-6-30 8:00:00'))->getTimestamp();
+		$start = (new DateTime('2024-7-1 04:00:00'))->getTimestamp(); // Start date with America/Toronto offset
+		$timezone = 'America/Toronto';
+		$sDT = (new DateTime('2024-7-1 04:00:00'))->getTimestamp();
+		;
+		$eDT = (new DateTime('2024-7-2 04:00:00'))->getTimestamp();
+		;
+
 		$apptConfg = new AppointmentConfig();
 		$apptConfg->setId(1);
 		$this->time->expects(self::once())
 			->method('getTime')
-			->willReturn($fakeFutureTimestamp);
-		$this->apptService->expects(self::never())
+			->willReturn($now);
+		$this->apptService->expects(self::once())
 			->method('findById')
-			->with(1);
-		$this->bookingService->expects(self::never())
-			->method('getAvailableSlots');
-		$this->logger->expects(self::once())
-			->method('warning');
+			->with(1)
+			->willReturn($apptConfg);
+		$this->bookingService->expects(self::once())
+			->method('getAvailableSlots')
+			->with($apptConfg, $sDT, $eDT);
 
-		$this->controller->getBookableSlots($apptConfg->getId(), $start, 'Europe/Berlin');
+		$this->controller->getBookableSlots($apptConfg->getId(), $start, $timezone);
+	}
+
+	public function testGetBookableSlotsTimezoneMinus10(): void {
+
```

Comment:
nit: empty first line of method body is unusual

Reply:
Corrected